docs/upgrade.html

```diff
@@ -30,6 +30,7 @@ <h5><a id="upgrade_320_notable" href="#upgrade_320_notable">Notable changes in 3
             Users can change this behavior to enable idempotence for some or all producers
             via Connect worker and/or connector configuration. Connect may enable idempotent producers
             by default in a future major release.</li>
+        <li>Kafka has replaced log4j and slf4j-log4j12 with reload4j and slf4j-reload4j. More info can be found at <a href"https://reload4j.qos.ch">reload4j</a>
```

Comment:
nit: Kafka has replaced log4j and slf4j-log4j12 with reload4j and slf4j-reload4j [to mitigate security risks]. ...

Reply:
Changed